sup3r/preprocessing/feature_handling.py

```diff
@@ -707,97 +704,8 @@ def compute(cls, data, height):
             Derived feature array
 
         """
-        _, v = transform_rotate_wind(
-            data[f'windspeed_{height}m'],
-            data[f'winddirection_{height}m'],
-            data['lat_lon'])
-        return v
-
-
-class UWindNsrdb(DerivedFeature):
```

Comment:
why dont we need these anymore? Dont we still need these if we want u/v from NSRDB instead of windspeed/direction? Not sure if we have a test for that.

Reply:
I added 'windspeed' -> 'wind_speed', and 'winddirection' -> 'wind_direction' in the feature registry for the solar data handler, so this enables use of UWind, VWind. I added a test in test_feature_handling also.

Reply:
https://github.com/NREL/sup3r/blob/bcac12a5496611e6376f1cceb3df001b99f237ab/sup3r/preprocessing/data_handling.py#L2095

Reply:
https://github.com/NREL/sup3r/blob/bcac12a5496611e6376f1cceb3df001b99f237ab/sup3r/preprocessing/data_handling.py#L2095:#L2098

Reply:
ok sweet!